docs/src/tutorials/linear_methods/discrete_time.md

```diff
@@ -47,7 +47,7 @@ V = vertices_list(X0) .|> Singleton |> UnionSetArray
 plot!(c)
 plot!(V)
 plot!(M(pi/4) * c) # rotate 45 degrees
-plot!(M(pi/4) * V)
+plot!(linear_map(M(pi/4), V))
```

Comment:
if this was meant to be explanatory only, for someone who is new to the library, using `*` seems more intuitive

Reply:
The question is whether the method really makes sense (lazy linear map of a union of singletons evaluates to the union of the concrete linear maps). I would argue this is not correct. If it is just for plotting, we should definitely support this, but in LazySets.

Reply:
I thought about it again and the solution is not so easy. Without this questionable method we get a `LinearMap{Matrix,UnionSetArray}`. The plot recipe for `LinearMap` uses epsilon-close approximation, which only works for nonconvex sets.
I think the best is to have a function to bring a set to a normal form, which is triggered by the plot recipe when the set is not convex. In this case it should recognize that `LinearMap{M,UnionSetArray{N,T}}` can be rewritten to `UnionSetArray{N,LinearMap{M,T}}`, and for unions there is a plot recipe.
An alternative is to rewrite to normal form by default when parsing `M * V`. But I would not do that, in part because it is not what you always want, may break assumptions, and is more expensive.

In any case, having a method here that only works for singletons does not make sense to me. It leads to the illusion that it always works and may also lead to wrong behavior in algorithms (suddenly the lazy linear map is concrete).

Reply:
ok, i missed the fact that `V` is a unionsetarray.

Reply:
I opened https://github.com/JuliaReach/LazySets.jl/issues/3475.